packages/fxa-settings/src/components/Modal/index.tsx

```diff
@@ -67,9 +67,9 @@ export const Modal = ({
           <div className="px-4 tablet:px-12 pb-10">
             <div>{children}</div>
             {hasButtons && (
-              <div className="flex mt-6">
+              <div className="flex justify-center mx-auto max-w-1xs">
```

Comment:
We might need that `mt-6` class: 
<img width="582" alt="image" src="https://user-images.githubusercontent.com/13018240/91913513-7073ef00-ec7b-11ea-8276-ce9a1f97285e.png">

Also I think this PR is a partial fix for #6327, I think when @meandavejustice's PR gets merged with the caution buttons we can close it?